plot_app/plotting.py

```diff
@@ -640,3 +649,91 @@ def _setup_plot(self):
         p.toolbar.logo = None
 
 
+class DataPlotSpec(DataPlot):
+    """
+    A spectrogram plot.
+    This does not do downsampling.
```

Comment:
This comment needs to be updated: it does downsample, just not dynamic

Reply:
ok, I've fixed the CI issue and updated the comment.

thank you again.